content/en/tracing/troubleshooting/_index.md

```diff
@@ -2,444 +2,109 @@
 
-```
-Max events per second reached (current=300.00/s, max=200.00/s). Some events are now being dropped (sample rate=0.54). Consider adjusting event sampling rates.
+1. [Tracer Startup Logs](#tracer-startup-logs)
 
-```
+    Startup logs are a great way to spot misconfiguration of the tracer, or the inability for the tracer to communicate with the Datadog Agent.  By comparing the configuration that the tracer sees to the one set within the application or container, we can identify any areas where a setting is not being properly applied.
 
+1. [Tracer Debug Logs](#tracer-debug-logs)
 
-To increase the APM rate limit for the Agent, configure the `max_events_per_second` attribute within the Agent's configuration file (underneath the `apm_config:` section). For containerized deployments (Docker, Kubernetes, etc.), use the `DD_APM_MAX_EPS` environment variable.
+    Tracer Debug logs go one step deeper than startup logs, and will help us to identify if integrations are instrumenting properly in a manner that we aren't able to necessarily check until traffic flows through the application.
 
-**Note**: Increasing the APM rate limit could result in increased costs for App Analytics.
+1. An [agent flare][4] from your agent in [debug mode][5]
 
+    Agent flares allow us to see what is happening within the Datadog Agent, if any traces are being rejected or malformed within the Agent.  This will not help if traces are not reaching the Agent, but does help us identify the source of an issue, as well as identifying any metric discrepancies.
 
-### Max connection limit
+1. A description of your environment
 
-If you encounter the following error message in your Agent logs, the default APM connection limit of 2000 has been exceeded:
+    Knowing how your application is deployed helps us identify likely issues for tracer-agent communication problems or misconfigurations.  For difficult issues, we may ask to a see a Kubernetes manifest or an ECS task definition, for example.
 
-```
-ERROR | (pkg/trace/logutil/throttled.go:38 in log) | http.Server: http: Accept error: request has been rate-limited; retrying in 80ms
-```
+1. Any automatic or [custom instrumentation][6], along with any configurations
 
+    Custom instrumentation can be a very powerful tool, but also can have unintentional side effects on your trace visualizations within Datadog, so we ask about this to rule it out as a suspect.  Additionally, asking for your automatic instrumentation and configuration allows us to confirm if this matches what we are seeing in both tracer startup and debug logs to look for discrepancies.
 
-To increase the APM connection limit for the Agent, configure the `connection_limit` attribute within the Agent's configuration file (underneath the `apm_config:` section). For containerized deployments (Docker, Kubernetes, etc.), use the `DD_APM_CONNECTION_LIMIT` environment variable.
+1. Versions of languages, frameworks, the Datadog Agent and Tracing Library being used
```

Comment:
I think convention would be to have a comma after "Datadog Agent" right before the "and".

Reply:
a nit pick i know 😿